sentry_sdk/tracing.py

```diff
@@ -695,8 +695,31 @@ def __repr__(self):
             )
         )
 
+    def _possibly_stated(self):
+        # type: () -> bool
+        """Returns whether the transaction might have been started.
+
+        If this returns False, we know that the transaction was not started
+        with sentry_sdk.start_transaction, and therefore the transaction will
+        be discarded.
+        """
+
+        # We must explicitly check self.sampled is False since self.sampled can be None
+        return self._span_recorder is not None or self.sampled is False
```

Comment:
`sampled=False` is a sign that the transaction has been started because it is set in `_set_inital_sampling_decision` that is only called in start_transaction, right?

Reply:
No, `sampled is False` tells us nothing about whether the transaction has been started – `self.sampled` could also be set in the `__init__` (i.e. if the user passes `sampled=False` to the constructor). However, if the user calls `start_transaction` on a transaction that has `sampled=False`, `self._span_recorder` will be `None`, even though the transaction was started with `start_transaction`.

As I attempted to explain in the docstring, this function's purpose is to tell us whether the transaction _might_ have been started with `start_transaction`. In other words:
  - If the function returns `True`, we _do not know_ whether the transaction was started with `start_transaction`, but it might have been.
      - Specifically, if `self.sampled is False`, we have no way to tell whether the user started the transaction with `start_transaction`, so we return `True`.
  - If the function returns `False`, we _know_ that the transaction _was not started_ with `start_transaction`.

Reply:
Got it, thanks for the explanation!